Rules/AvoidUsingInternalURLs.cs

```diff
@@ -44,6 +44,30 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
             {
                 foreach (StringConstantExpressionAst expressionAst in expressionAsts)
                 {
+                    //Check if XPath is used. If XPath is used, then we don't throw warnings.
+                    Ast parentAst = expressionAst.Parent;
+                    IEnumerable<Ast> invocation = parentAst.FindAll(test => test is InvokeMemberExpressionAst, true);
```

Comment:
Should we just check that the parentAst is a InvokeMemberExpressionAst? I don't think there would be more than 1 InvokeMemberExpressionAst that we will be looking at?